cdap-common/bin/functions.sh

```diff
@@ -207,13 +207,7 @@ cdap_stop_pidfile() {
 #
 cdap_check_pidfile() {
   local readonly __pidfile=${1} __label=${2:-Process}
-  if [[ -f ${__pidfile} ]]; then
-    local readonly __pid=$(<${__pidfile})
-    if [[ $(cdap_status_pidfile ${__pidfile} ${__label} >/dev/null) ]]; then
-      echo "${__label} running as PID ${__pid}. Stop it first."
```

Comment:
i was thinking leave the ``cdap_status_pidfile`` output (I'm assuming its called when you do "cdap status"), but just add an extra line for this case to indicate "already started, did you mean restart?".  Weird I know... i'm on the fence as well, but i can imagine someone not realizing its stopped, running start and thinking they have just restarted.